BridgeSDK/Networking/SBBNetworkManager.h

```diff
@@ -212,6 +213,24 @@ typedef NS_ENUM(NSInteger, SBBEnvironment) {
 @end
 
 
+/**
```

Comment:
I don't feel like this is the right place to put this, either. AppCore does access SBBNetworkManager directly, unlike SBBBridgeNetworkManager, but only when using it to access other (non-Bridge) REST APIs so... yeah.

Maybe this should just exist in its own header, SBBBridgeAppDelegate.h...